build/cluster.js

```diff
@@ -45,6 +45,13 @@ const kubernetesArchive = 'kubernetes.tar.gz',
  */
 let clusterProcess = null;
 let exec = childProcess.exec;
+/**
+ * A Number, representing the ID value of the timer that is set for function which periodically
+ * checks if cluster is running.
+ *
+ * @type {number}
```

Comment:
The type should be `{?number}`, because number is a built in that is is always non null, but you assign null to it. Hence the `?`.